pkg/datasources/nats/datasource.go

```
@@ -285,7 +287,7 @@ func (s *KeyValueSource) processUpdates(ctx context.Context, watcher nats.KeyWat
 			}
 			entryBytes, err := json.Marshal(entry)
 			if err != nil {
-				return err
+				return
```

Is it safe to swallow this error?

---

totally safe